beets/autotag/hooks.py

```diff
@@ -88,6 +88,7 @@ def __init__(self, tracks, album=None, album_id=None, artist=None,
         self.tracks = tracks
         self.asin = asin
         self.albumtype = albumtype
+        self.albumtypes = albumtypes
```

Comment:
There's actually no need to add `albumtypes` as an explicit parameter here; it can go into `**kwargs`. We have a bunch of fields listed here purely for legacy reasons; the `AlbumInfo` class is now "flexible" instead of baking in a fixed set of fields.

Reply:
Gotcha! I've removed what I had added here, seems to work still!